frontend/packages/rhoas-plugin/src/topology/createConnector.ts

```diff
@@ -8,8 +9,12 @@ const createServiceBindingConnection = (source: Node, target: Node) => {
   return createServiceBinding(sourceResource, targetResource).then(() => null);
 };
 
-export const getCreateConnector = (createHints: string[]) => {
-  if (createHints) {
+export const getCreateConnector = (createHints: string[], source: Node, target: Node) => {
```

Comment:
we could add a test for this util to capture the conditions under which returns.